Gemfile.lock

```diff
@@ -452,7 +452,7 @@ DEPENDENCIES
   will_paginate
 
 RUBY VERSION
-   ruby 2.4.2p198
+   ruby 2.4.9p362
 
 BUNDLED WITH
-   1.17.1
+   1.17.2
```

Comment:
#### Minor correction
I notice that you've changed the bundler version from 1.17.1 to 1.17.2 was this deliberate? Bundler releases of [1.17.2 and 1.17.3 were both very minor](https://github.com/rubygems/bundler/blob/master/CHANGELOG.md#1173-2018-12-27). I'm not bothered but if intentional it should be noted somewhere in the commit.

Reply:
Good spot, this was not intentional